source-code/device.lua

```diff
@@ -1,6 +1,6 @@
 local me = {
   name = "Security",
-  hwVersion = "2.0.beta1",
-  swVersion = "2.0.beta1"
+  hwVersion = "2.0.0",
+  swVersion = "2.0.0"
```

Comment:
Can we make this `2.0.beta2` for now?
I want to release one beta version to the open-source community today and then we can release a 2.0.0 tonight or tomorrow when I have the new app icons from Matt.